src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
@stipsan 👌 I've looked into covering this with test and I see it requires mocking the PHP functions. It can be done, but needs some thought to not make a mess. I'll have to do it later.